password_policies/south_migrations/0001_initial.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding model 'PasswordChangeRequired'
+        db.create_table(u'password_policies_passwordchangerequired', (
+            (u'id', self.gf('django.db.models.fields.AutoField')(primary_key=True)),
+            ('created', self.gf('django.db.models.fields.DateTimeField')(auto_now_add=True, db_index=True, blank=True)),
+            ('user', self.gf('django.db.models.fields.related.OneToOneField')(related_name='password_change_required', unique=True, to=orm['auth.User'])),
+        ))
+        db.send_create_signal(u'password_policies', ['PasswordChangeRequired'])
+
+        # Adding model 'PasswordHistory'
+        db.create_table(u'password_policies_passwordhistory', (
+            (u'id', self.gf('django.db.models.fields.AutoField')(primary_key=True)),
+            ('created', self.gf('django.db.models.fields.DateTimeField')(auto_now_add=True, db_index=True, blank=True)),
+            ('password', self.gf('django.db.models.fields.CharField')(max_length=128)),
+            ('user', self.gf('django.db.models.fields.related.ForeignKey')(related_name='password_history_entries', to=orm['auth.User'])),
+        ))
+        db.send_create_signal(u'password_policies', ['PasswordHistory'])
+
+    def backwards(self, orm):
+        # Deleting model 'PasswordChangeRequired'
+        db.delete_table(u'password_policies_passwordchangerequired')
+
+        # Deleting model 'PasswordHistory'
+        db.delete_table(u'password_policies_passwordhistory')
+
```

Comment:
To be honest, I've only added south migrations primarily for selfish reasons in that the application I'm using this on is < 1.7 and we use South for our migrations. But thought it might help out others as well.

I can happily make the changes for custom user models, but I can't say I've seen handling custom groups in any of the south migrations I've look at. But if you know of any / how to handle them point me in the right direction and I'll add those as well